docker-compose.yml

```diff
@@ -18,4 +15,13 @@ services:
       KEYCLOAK_ADMIN_PASSWORD: 'admin'
```

Comment:
I don't know if it's easily possible, but could we have this Keycloak instance automatically start up with a configured realm for local development? 
https://www.keycloak.org/server/containers#_importing_a_realm_on_startup

Reply:
Good call! I exported my local setup that just has a basic realm with postman as a client and the container will now pull that in from a volume on startup.